HintList.py

```diff
@@ -1599,6 +1599,7 @@ def tokens_required_by_settings(world):
     'bridge_hearts':                                            ("hearts", None, 'bridge'),
 
     'ganonBK_dungeon':                                          ("hidden somewhere #inside its castle#", None, 'ganonBossKey'),
+    'ganonBK_regional':                                       ("hidden somewhere #inside or nearby its castle#", None, 'ganonBossKey'),
```

Comment:
```suggestion
    'ganonBK_regional':                                         ("hidden somewhere #inside or nearby its castle#", None, 'ganonBossKey'),
```

Reply:
The spacing will be fixed next time I push a commit. Thanks for catching it.